inc/features/blocks/personalization-variant/edit.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+const { InnerBlocks } = wp.blockEditor;
+const { compose } = wp.compose;
+const { withSelect, withDispatch } = wp.data;
+
+const Edit = ( {
+	hasChildBlocks,
+	isSelected,
+	selectParent,
+} ) => {
+	// Autofocus experience block variant parent on select.
+	if ( isSelected ) {
+		selectParent();
```

Comment:
This seems like it could render thrash; should be in a `useEffect( ..., [] )` I think